sip/headers.go

```diff
@@ -121,6 +127,10 @@ func (hs *headers) unref(header Header) {
 		hs.contentType = nil
 	case *MaxForwardsHeader:
 		hs.maxForwards = nil
+	case *ReferToHeader:
```

Comment:
Fast references are only for headers that you need to have or read. 
Please remove this

Reply:
removed statement from both setHeaderRef and unref